oioswift/server.py

```diff
@@ -95,9 +95,8 @@ def __init__(self, conf, memcache=None, logger=None, account_ring=None,
 
         self.storage = storage or \
             ObjectStorageApi(sds_namespace, endpoint=sds_proxy_url, **sds_conf)
-        # self.delete_slo_parts = \
-        #     config_true_value(conf.get('delete_slo_parts', True))
-        self.delete_slo_parts = False
+        self.delete_slo_parts = \
+            config_true_value(conf.get('delete_slo_parts', True))
```

Comment:
Should we let default value to `False` and let ops make decision regarding this feature ?